assets/app/view/game/dividend.rb

```diff
@@ -135,13 +135,18 @@ def cleanup
       def render_variable(entity)
         max = (@step.variable_max(entity) / entity.total_shares).to_i
 
-        input = h(:input, style: { margin: '1rem 0px', marginRight: '1rem' }, props: {
-          value: max,
-          min: 0,
-          max: max,
-          type: 'number',
-          size: max.to_s.size,
-        })
+        input = h(:input,
```

Comment:
all these should be new lines
h(
  :input,
..

Reply:
Multiline could be enforced by https://docs.rubocop.org/rubocop/1.7/cops_layout.html#layoutmultilinemethodargumentlinebreaks, but that would also affect hundreds of other cases were this wouldn’t be desirable.
I haven’t found a cop to enforce a newline on multiline method class, especially not limited to only h().